src/main/resources/db/migration/V202409081400__create_pets.sql

```diff
@@ -0,0 +1,31 @@
+
+CREATE TABLE pets
```

Comment:
better be consistent and name the table pet in singular (or name the owner table owners in plural)